### ISSUE TITLE
Incorrect seldon-core package version in 1.1.0 builder image

### ISSUE THREAD

**Issue by groszewn** [SeldonIO/seldon-core#1718]
The `seldonio/seldon-core-s2i-python37:1.1.0` builder image contains the incorrect version of the seldon-core python pacakge.

```
docker run -it seldonio/seldon-core-s2i-python37:1.1.0 bash
(base) root@0c5919451d38:/microservice# pip list | grep seldon-core
seldon-core            1.0.3-SNAPSHOT /microservice/python
```

**Comment by RafalSkolasinski** [comment]
This seems to come from a hardcoded value in `/microservice/python/seldon_core/version.py`
```
$ docker run --rm -it seldonio/seldon-core-s2i-python37:1.1.0
(base) root@0c33072416fc:/microservice# cat /microservice/python/seldon_core/version.py
__version__ = "1.0.3-SNAPSHOT"
```
Maybe version there didn't get updated before building `1.1.0` image.

I inspected the code inside the image and it seems to have `1.1.0` fixes (at least the one I was working on as them I remember) to is is probably only a problem of wrong version string reported to `pip`.

**Comment by cliveseldon** [comment]
Linked to https://github.com/SeldonIO/seldon-core/issues/1732

**Comment by cliveseldon** [comment]
There is an `update_package` in the python Makefile. It should be called as part of the release.py
https://github.com/SeldonIO/seldon-core/blob/master/release.py

**Comment by cliveseldon** [comment]
Check fixed by #1775
